Ical.Net/Evaluation/TodoEvaluator.cs

```diff
@@ -67,7 +65,7 @@ private static void DetermineStartingRecurrence(IEnumerable<CalDateTime> rdate,
 
     private void DetermineStartingRecurrence(RecurrencePattern recur, ref CalDateTime referenceDateTime)
     {
-        if (recur.Count.HasValue)
+        if (recur.Count.HasValue && Todo.Start != null)
```

Comment:
This whole `EvaluateToPreviousOccurrence` that this method belongs to, is pretty weird. Not sure, what the right behavior in this case is. Probably just return if Todo.Start is null. Going into the else branch if its null probably isn't correct.